wiki.md

```diff
@@ -73,11 +75,17 @@ To connect to the payment gateway:
 - Click on **Create Page** button **_(mandatory)_**
 - Click on **Save changes** button (in the bottom of the page).
 
-To select the Terminals:
+This will populate all the available terminals for the currenct API user.
+If you face any issue click on **Refresh connection** button, this will fetch the terminals again and populate the list.
+
+## Synchronize payment methods
+
+To synchronize the terminals with the gateway, click on the **Synchronize payment methods** button. This will fetch the latest terminals from the gateway and will automatically configure based on the store country.
```

Comment:
We should also add the explanation on how to do this using the `terminal-config/altapay_config.php` file, for companies that want to automate this at provision/deploy.